File: SecondLabPython/OOP.cs
public class People
{
    protected string p_Name = "";
    protected int p_Age = 0;

    public People(string name, int age) {
        this.p_Name = name;
        this.p_Age = age;
    }

    public void sayHi() {
        Console.WriteLine($"{this.p_Name} говорит привет");
    }
    public void sayAge()
    {
        Console.WriteLine($"{this.p_Name} {this.p_Age} лет");
    }
}

public class Denis : People
{
    private string p_Group = "";

    public Denis(string name, int age, string group) : base(name, age)
    {
        this.p_Group = group;
    }

    public void sayGroup()
    {
        Console.WriteLine($"{this.p_Name} учится в группе {this.p_Group}");
    }
}

public class Program
{
    public static void Main()
    {
        Denis denis = new Denis("Denis", 18, "ИП-391к");

        denis.sayHi();
        denis.sayAge();
        denis.sayGroup();
    }
}

File: SecondLabPython/OOP.py
class People: 
	def __init__(self, name, age):
		self.name = name
		self.age = age

	def sayHi(self):
		print(f"{self.name} говорит привет")

	def sayAge(self):
		print(f"{self.name, self.age}")


class Denis(People):
		def __init__(self, name, age, group):
			super().__init__(name,age)
			self.group = group
				
		def sayGroup(self):
			print(f"{self.name} учится в группе {self.group}")


denis = Denis('Denis', 18, 'ИП-391к')

denis.sayHi()
denis.sayAge()
denis.sayGroup()